std/parallelism.d

```diff
@@ -984,6 +988,10 @@ uint totalCPUsImpl() @nogc nothrow @trusted
         {
             auto nameStr = "hw.ncpu\0".ptr;
         }
+        else version(DragonFlyBSD)
+        {
+            auto nameStr = "hw.ncpu\0".ptr;
```

Comment:
Off topic: this looks like a very old style. IIRC are strings literals always zero-terminated?

Reply:
Don't know, maybe someone else can answer this one.

Reply:
Yes, that's true, the string literals are always zero-terminated. I was wondering if something is also doing `nameStr.length` in which case it makes sense putting this `\0` but it doesn't look like it.

Reply:
@nemanja-boric-sociomantic 
So should i rewrite this to: ```auto nameStr = "hw.ncpu".ptr;```, or just leave it as is.

The surrounding lines are doing the same, though. and i do not want to correct them (Only DragonFlyBSD related lines should be touched in this PR).

Reply:
I would leave them as is IMHO and do this separately.